doc/developer-guides/hld/hv-virt-interrupt.rst

```diff
@@ -224,22 +243,24 @@ Virtual Interrupt Injection
 The source of virtual interrupts comes from either DM or assigned
 devices.
 
--  **For SOS assigned devices**: as all devices are assigned to SOS
-   directly. Whenever there is a device's physical interrupt, the
-   corresponding virtual interrupts are injected to SOS via
-   vLAPIC/vIOAPIC. SOS does not use vPIC and does not have emulated
-   devices. See :ref:`device-assignment`.
+-  **For Service VM assigned devices**: as all devices are assigned to
+   Service VM directly. Whenever there is a device's physical interrupt,
+   the corresponding virtual interrupts are injected to Service VM via
```

Comment:
the beginning of the sentence is "a interrupt", so the virtual interrupts should be interrupt?